default/python/AI/TechsListsAI.py

```diff
@@ -467,7 +455,7 @@ def __init__(self):
             self.hull,
             self.misc,
             self.economy,
-            self.hull,
```

Comment:
TechGroup3 have asteroid reformation, heavy asteroid hulls and contragravitational maintenance techs in the self.hull list. That is not very effective. Asteroid reformation gets researched very soon. Heavy asteroid hulls are currently not a great deal compared to normal asteroid hulls. IMHO if one researches scattered asteroid hulls, heavy asteroid hull get automatically researched as well - so personally I would rather drop the heavy asteroid hulls  from TechGroup3 as well or maybe switch order for contragrav and heavy asteroid hulls and not remove this self.hull reference, but the one before (so heavy asteroid hulls are quite late to be researched).

Reply:
so, instead one could remove the entry in line 467